BrocoliSe/BrocoliSeTests/Mocks/DiarySceneMock.swift

```diff
@@ -10,6 +10,8 @@ import CoreData
 @testable import BrocoliSe
 
 class DiarySceneMock: DiarySceneDelegate {
+
+    
```

Comment:
lint: Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. (vertical_whitespace)